src/services/paras/ParasService.ts

```diff
@@ -468,6 +468,10 @@ export class ParasService extends AbstractService {
 		 * A sample can be represented by `afterEarlyEnd` / `sampleLength`.
 		 * When we are in the endingPeriod, the offset is represented by:
 		 * `sample - 1`.
+		 *
+		 * If the current phase is in `vrfDelay`, and you are interested in
+		 * querying the winners of the auction that just finished, it is advised
+		 * to query the last block of the `endingPeriod`.
```

Comment:
This should go in open api docs, if it isn't already

Reply:
Yea I have it locally, just havent commited it yet.

Reply:
its up now.